tests/server/test_server_validation.py

```diff
@@ -5,7 +5,7 @@
 
 from optimade.validator import ImplementationValidator
 
-from .utils import SetClient
+from .utils import SetClient, TestClient
```

Comment:
That's cheap!
```suggestion
from fastapi.testclient import TestClient

from .utils import SetClient
```

Reply:
But actually, why would you need to import this?
```suggestion
from .utils import SetClient
```

Reply:
Ah, leftover from testing, weird that flake8 didn't complain in the checks...

Reply:
Yikes, I just expanded the flake8 tab in the CI and it is screaming at us quite a lot, didn't realise we were ignoring minor errors it by default :)

Reply:
Hmmmmm